test/e2e/specs/liquidation.spec.js

```diff
@@ -54,6 +60,7 @@ describe('Wallet App Test Cases', () => {
       });
       cy.setOraclePrice(12.34);
     });
+    /* eslint-disable ui-testing/missing-assertion-in-test */
```

Comment:
this is legit, right? perhaps you're testing that the function executes without throwing. does Cypress have something like Ava's `t.pass();` for if the line is reached?

Reply:
Didn't find anything like `t.pass()` :D